types/index.d.ts

```diff
@@ -289,10 +289,61 @@ declare namespace PhonegapPluginPush {
 		from?: string
 		notId?: string
 	}
-
+  /**
+   * Set Channel Variable,once created cannot be changed programmtically for importance and vibration
+   */
+  interface channel{
+    /**
+     * channel ID
```

Comment:
We can copy the docs on this from the official API docs.
In this case, it would be:
```suggestion
     * The id of the channel. Must be unique per package. The value may be truncated if it is too long.
```